main/output-ctags.c

```diff
@@ -131,8 +131,8 @@ static int addExtensionFields (MIO *mio, const tagEntryInfo *const tag)
 	{
 		const char* k = NULL, *v = NULL;
 
-		k = escapeName (tag, FIELD_SCOPE_KIND_LONG);
-		v = escapeName (tag, FIELD_SCOPE);
+		k = escapeFieldValue (tag, FIELD_SCOPE_KIND_LONG);
+		v = escapeFieldValue (tag, FIELD_SCOPE);
 
```

Comment:
This is needed to make FQN, full qualified tags.
However, this should be done in common routine, not in output-*.c.

I will move this code for FQN making to caller side when merging json-output code.